src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -215,6 +223,136 @@ public function exportAction(OrderFilters $filters)
             ->setFileName('order_' . date('Y-m-d_His') . '.csv');
     }
 
+    public function viewAction(int $orderId): Response
```

Comment:
`AdminSecurity` to add in a following PR

Reply:
Already part of TODO list.